frontend/src/components/SearchLozenge/SearchLozenge.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './SearchLozenge.css';
+
+const SearchLozenge = (props) => {
+  let { searchTerm } = props;
+  if (!searchTerm) {
+    searchTerm = '';
+  }
+  const lozenges = searchTerm.split(' ');
+  return (
+    <div className="searchLozenge container">
+      <div className="row">
+        <div className="lozengeBubbles">
+          {lozenges.map(lozenge => (
+            <div className="lozenges">
+              {lozenge.charAt(0).toUpperCase() }
+              {lozenge.substr(1)}
+              {' '}
+              {' x'}
```

Comment:
I assume you're working on it, but definitely using a link or button or something here is gonna be worthwhile.